### PR TITLE
Decouple SetGoverningApoJob from Fedora 3

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Review by justinlittman — APPROVED** [review]

File: spec/jobs/set_governing_apo_job_spec.rb [hunk 1]
```diff
@@ -96,34 +96,30 @@
...
+        let(:object_client1) { instance_double(Dor::Services::Client::Object, find: cocina1, update: true) }
         let(:object_client3) { instance_double(Dor::Services::Client::Object, find: cocina3) }
 
         before do
           allow(Dor::Services::Client).to receive(:object).with(pids[0]).and_return(object_client1)
           allow(Dor::Services::Client).to receive(:object).with(pids[1]).and_raise(Dor::Services::Client::NotFoundResponse)
           allow(Dor::Services::Client).to receive(:object).with(pids[2]).and_return(object_client3)
 
-          allow(Dor).to receive(:find).with(pids[0]).and_return(item1)
+          # allow(Dor).to receive(:find).with(pids[0]).and_return(item1)
```
Comment (aaron-collier) [inline spec/jobs/set_governing_apo_job_spec.rb:107]: Do we want to keep these comments for now or cruft?

File: spec/jobs/set_governing_apo_job_spec.rb [hunk 2]
```diff
@@ -96,34 +96,30 @@
...
           allow(subject).to receive(:check_can_set_governing_apo!).with(cocina1, state_service).and_return true
           allow(subject).to receive(:check_can_set_governing_apo!).with(cocina3, state_service).and_raise('user not allowed to move to target apo')
 
-          allow(Dor).to receive(:find).with(new_apo_id).and_return(apo)
+          # allow(Dor).to receive(:find).with(new_apo_id).and_return(apo)
```
Comment (aaron-collier) [inline spec/jobs/set_governing_apo_job_spec.rb:111]: Same as above?

File: spec/jobs/set_governing_apo_job_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/jobs/set_governing_apo_job_spec.rb]: It's cruft. I've removed.

File: spec/jobs/set_governing_apo_job_spec.rb (hunk 2 above)
Comment (jcoyne) [reply spec/jobs/set_governing_apo_job_spec.rb]: It's cruft. I've removed.

**Review by aaron-collier — APPROVED** [review]

**Pull request merged**